### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Khopa**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+* Report bugs by opening issues here on Github.
+* Help others users on Discord by answering their questions.
+* Raise awareness about the project, by making a video and/or a tutorial.
+
+Should you report a bug, please use the search bar at the top of the page to see if it has already been reported.
+Note that you may need to remove the filter for open bugs if it's something we've recently fixed.
+
+## Making content for Liberation
+
+You can create new campaigns setup : See [campaign creation wiki](https://github.com/Khopa/dcs_liberation/wiki/Custom-Campaigns).
```
Comment (DanAlbert) [inline CONTRIBUTING.md]: Phrasing here is a little odd. Drop the word setup.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+You can also improve existing campaigns.
+
+You can then submit new campaigns on the "campaigns" channel on Discord, or by making a pull request if you are comfortable with git. 
+
+## Develop new features
+
+If you want to develop a new feature, we recommend you first open an issue describing the new feature and discuss it before starting development.
```
Comment (DanAlbert) [inline CONTRIBUTING.md]: And join discord :)

**Review by DanAlbert — CHANGES_REQUESTED** [review]
Couple minor improvements.. otherwise looks good.

File: CONTRIBUTING.md (hunk 1 above)
Comment (Khopa) [reply CONTRIBUTING.md]: Thanks, removed the word

File: CONTRIBUTING.md (hunk 2 above)
Comment (Khopa) [reply CONTRIBUTING.md]: 👍

**Pull request merged**